Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ruby:2.7-alpine as build
+FROM ruby:3.0-alpine as build
```

Comment:
Just wondering, is there any reason to not use `ruby:3.0.2-alpine` or better `ruby:3.0.2-alpine3.14` directly? I can submit PR later to lock base image to particular one, since this one could be updated using the same tag (with both Ruby patch version or alpine one).

Reply:
we don't pin patch versions, since we are assuming that they won't have any breaking change. this way we would be able to update automatically (without code change), useful for security releases. we should update our doc to only mention major.monor version.